src/makeCalculator.js

```diff
@@ -4,7 +4,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+  const calculator = {
+    operate(callback, number) {
+      if (typeof callback === 'function') {
+        result = callback(result, number);
+      }
+
+      return this;
+    },
+
+    getResult() {
+      return result;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      if (b !== 0) {
+        return a / b;
+      } else {
+        return a;
+      }
+    },
```

Comment:
The `divide` method should handle division by zero more robustly. Currently, it returns `a` when `b` is zero, which might not be the best approach. Consider throwing an error or returning a specific value to indicate division by zero.